horovod/common/ops/nccl_operations.h

```diff
@@ -19,6 +20,9 @@
 
 #if HAVE_CUDA
 #include <nccl.h>
+#if NCCL_VERSION_CODE >= 2700
```

Comment:
Should we use the [NCCL_VERSION](https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/api/comms.html#c.ncclGetVersion) macro here?

```
if NCCL_VERSION_CODE >= NCCL_VERSION(2, 7, 0)
```

Reply:
Yes, this seems more fool-proof. Made the change. On a related note, I do still use `2700` explicitly for skipping tests based on the output of `nccl_built()`.